graphql/resolvers.go

```diff
@@ -1730,6 +1730,13 @@ func (r *queryResolver) PatchTasks(ctx context.Context, patchID string, sorts []
 			taskSorts = append(taskSorts, task.TasksSortOrder{Key: key, Order: order})
 		}
 	}
+	v, err := model.VersionFindOneId(patchID)
```

Comment:
Could improve this by doing a FindOne(byId.WithFields(requester)) since we don't need the whole version.